datasetinsights/estimators/base.py

```diff
@@ -33,12 +36,14 @@ def create_estimator(
     # todo this makes it so that we lose the tensorboard
     #  writer of non-master processes which could make debugging harder
 
+    writer = SummaryWriter(log_dir=tb_log_dir)
+
     kfp_writer = KubeflowPipelineWriter(
         tb_log_dir=tb_log_dir,
-        filename=kfp_metrics_filename,
-        filepath=kfp_metrics_dir,
+        kfp_log_dir=kfp_log_dir,
+        kfp_metrics_filename=kfp_metrics_filename,
+        kfp_ui_metadata_filename=kfp_ui_metadata_filename,
```

Comment:
What's the difference between `kfp_metrics_file` and `kfp_ui_metadata_file`?

Reply:
`kfp_metrics_file`: https://www.kubeflow.org/docs/pipelines/sdk/pipelines-metrics/
`kfp_ui_metadata_file `: https://www.kubeflow.org/docs/pipelines/sdk/output-viewer/#tensorboard